package.json

```diff
@@ -1,100 +1,94 @@
 {
   "name": "@payfit/multi-semantic-release",
-  "author": "Dave Houlbrooke <dave@shax.com>",
-  "version": "3.16.0-alpha.22",
+  "author": "PayFit",
+  "version": "0.0.0-development",
```

Comment:
```suggestion
  "version": "0.0.0-semantic-release",
```
instead ?

Reply:
Don't know, seems like it is the new practice of semrel as it is the one than generated this for me 🙈

Reply:
Oh ok, let's keep it then